modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CreateTableParams.java

```diff
@@ -20,67 +20,51 @@
 import java.util.List;
 import org.jetbrains.annotations.Nullable;
 
-/**
- * CREATE TABLE statement.
- */
+/** CREATE TABLE statement. */
 public class CreateTableParams extends AbstractTableCommandParams {
     /** Creates parameters builder. */
     public static Builder builder() {
         return new Builder();
     }
 
+    private CreateTableParams() {
+        // No-op.
+    }
+
     /** Primary key columns. */
-    @Nullable
     private List<String> pkCols;
 
     /** Colocation columns. */
-    @Nullable
     private List<String> colocationCols;
 
-    /** Columns. */
-    private List<ColumnParams> cols;
+    /** Columns, {@code null} if not set. */
+    private @Nullable List<ColumnParams> cols;
```

Comment:
When can it be null? Is it possible to create a table without any columns?

Reply:
fix it